--- a/CMakeLists.txt	2024-03-24 15:17:52.758127166 +0100
+++ b/CMakeLists.txt	2024-03-24 15:18:00.230138348 +0100
@@ -58,7 +58,7 @@
 
 #--------------------------------------
 # Find gz-rendering
-gz_find_package(gz-rendering VERSION 10 REQUIRED OPTIONAL_COMPONENTS ogre ogre2)
+gz_find_package(gz-rendering VERSION 10 REQUIRED OPTIONAL_COMPONENTS ogre2)
 
 if (TARGET gz-rendering::ogre)
   set(HAVE_OGRE TRUE)
