--- a/CMakeLists.txt	2024-03-24 15:17:52.758127166 +0100
+++ b/CMakeLists.txt	2024-03-24 15:18:00.230138348 +0100
@@ -61,7 +61,7 @@
 
 #--------------------------------------
 # Find gz-rendering
-gz_find_package(gz-rendering9 REQUIRED OPTIONAL_COMPONENTS ogre ogre2)
+gz_find_package(gz-rendering9 REQUIRED OPTIONAL_COMPONENTS ogre2)
 set(GZ_RENDERING_VER ${gz-rendering9_VERSION_MAJOR})
 
 if (TARGET gz-rendering${GZ_RENDERING_VER}::ogre)
